src/workerd/api/actor-destroy-test.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023 Cloudflare, Inc.
+// Licensed under the Apache 2.0 license found in the LICENSE file or at:
+//     https://opensource.org/licenses/Apache-2.0
+
+import * as assert from 'node:assert'
+
+export class DurableObjectExample {
+  constructor(state, env) {
+    this.state = state;
+  }
+
+  async fetch() {
+    assert.ok(this.destroyCalled);
```

Comment:
Is it OK that events can be delivered after deletion? I would think it's important that the object is not permitted to receive any more events after deletion since if it wrote any data in the event, that data won't have a chance to be migrated by the destroy handler.

Reply:
Yeah, generally in agreement. The object should be reset.